app/web/features/auth/signup/Signup.tsx

```diff
@@ -126,6 +126,9 @@ function CurrentForm() {
         <Typography variant="h1" gutterBottom>
           {t("auth:feedback_form.header")}
         </Typography>
+        <Typography variant="h1" gutterBottom>
+        {t("auth:feedback_form.skiptitle")}
```

Comment:
This doesn't actually link to anything to let you skip the step at the moment. You'll need to add a `<Link>` element that more or less clone the behaviour of clicking the submit button, and do this with [the `Trans` component](https://react.i18next.com/latest/trans-component) so the link is interpolated correctly from the JSON resource.

Reply:
I just went through the codebase of what submit button does and it does state changing and it doesn't not do route changing!.So what should I do? Shall I route it to login page instead?